addresses/Addresses.sol

```diff
@@ -331,6 +331,22 @@ contract Addresses is IAddresses, Test {
         }
     }
 
+    /// @notice add or change an address (if is aready set) for the current chainId
+    /// @param name the name of the address
+    /// @param addr the address to change to
+    /// @param isContract whether the address is a contract
+    function addOrChangeAddress(
```

Comment:
For running integration tests in a fork simulation, we still want to deploy the proposal contracts even if they have already been deployed. Otherwise, what is the point of having integration tests if they always run against an already executed proposal?

Reply:
Ok, now it makes sense to me to revert that. I was concerned if we used the deployed contracts, we wouldn't be testing the entire proposal because we wouldn't be testing the deploy function. But validating the deploy function shouldn't be in the integration tests, we should have unit tests for that. And then in the examples repo we have integration tests that only deploy contracts if has not already been deployed